doc/decoding/decoding_intro.rst

```diff
@@ -198,9 +198,7 @@ The prediction objects have two important methods:
 
 .. warning::
 
-    **Do not predict on data used by the fit:** the prediction that we obtain
-    here is to good to be true (see next paragraph).
-    Here we were just doing a sanity check.
+    **Do not predict on data used by the fit:**
```

Comment:
It is true that this comment is not very clear at that place. 

Let me suggest: "**Do not predict on data used by the fit: this would yield misleadingly optimistic scores.**"